packages/astro/package.json

```diff
@@ -114,7 +114,7 @@
     "test:e2e:match": "playwright test -g"
   },
   "dependencies": {
-    "@astrojs/compiler": "^2.5.0",
+    "@astrojs/compiler": "0.0.0-mk-slot-expr-fix-20240130083013",
```

Comment:
We don't need to update the compiler version here, this was changed just to publicly validate the compiler bug fix.